tests/hamcrest_unit_test/core/raises_with_parens_test.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
```

Comment:
Don't create an entire separate unit test here; this can be added to the existing raises test suite

Reply:
I've move the test class into the existing test file. The existing raises tests